common/sgx/verifier.c

```diff
@@ -250,6 +256,16 @@ static oe_result_t _fill_with_known_claims(
         sgx_report_body = &((sgx_quote_t*)report_body)->report_body;
         OE_CHECK(
             oe_parse_sgx_report_body(sgx_report_body, true, &parsed_report));
+
+        sgx_quote = (sgx_quote_t*)report_body;
+        pcesvn = (uint16_t) & (sgx_quote)->pce_svn;
```

Comment:
This results in getting the last 16-bit address of the pce_svn. Can you double check?

Reply:
I verified, the & is needed to correctly access the structure's components

Reply:
I'm still curious about this: `sgx_quote` is a pointer. You should be able to access `pcv_svn` by just using `sgx_quote->pcv_svn`. `&sgx_quote->pcv_svn` is effectively the address of the `pcv_svn` within the struct. Why do you need to get the address (and only 16-bit given it's casted) instead of the value?

Reply:
Fixed this bug. Thank you for pointing out @mingweishih :-)